### ISSUE TITLE
nothing appears in my video or audio plugins

### ISSUE THREAD

**Issue by MatthewHunter1983** [project64/project64#538]
as of new builds since about a month or two ago, video and audio plugins are not present ... why? and older ones don't work. and how do I get a game to run? extract to project64 2.2 install directory? completely confused/frustrated. thanks!

**Comment by AmbientMalice** [comment]
You aren't creating 64 bit builds, by any chance?

**Comment by MatthewHunter1983** [comment]
probably not :\ no idea what that means. can some1 plz explain? I just want to play a game  (which I legally own) on a newer revision of project64 and continue to provide feedback here. sorry for the newb question. how do I make video/audio plugins appear and run a game?

**Comment by MatthewHunter1983** [comment]
there are no audio/video plugins even in the Plugin64 folder. and when I put some in there they don't appear in the plugins menu in project64. do I install 2.2 to 64 bit programs folder? do I extract the latest git to that folder?

**Comment by MatthewHunter1983** [comment]
just tried all of those things to no avail. can someone please just tell me the two-step process to load plugins and run a game w/ the 64 bit build? i don't know what create a 64 bit build means. never had this much trouble running this git. :\ sorry for the trouble. thanks!

**Comment by MatthewHunter1983** [comment]
1 last piece of seemingly relevant info: so i tried 32 bit version of project64 git and it works fine. gfx sound and even 1.6 plugins appear in the "Plugin" folder. 64 bit version none of those folders exist. does not work when I copy them over. what am I missing in 64 bit version? would like to try it.

**Comment by LegendOfDragoon** [comment]
@MatthewHunter1983 64bit is incomplete at this point in time. Just use 32bit for now. Please close this issue :smile: .

The 64bit exe loads plugins from a folder called Plugin64 iirc. There aren't many 64bit plugins available for zilmar spec.

**Comment by cxd4** [comment]
@MatthewHunter1983 Don't close this issue.  64-bit is 100% complete, just not merged officially to the primary repository yet.

**Comment by LegendOfDragoon** [comment]
>64-bit is 100% complete

Unhandled Memory Actions need to be handled.

**Comment by cxd4** [comment]
I have no idea what you just said.

I'm not implementing MemoryFilter.

**Comment by LegendOfDragoon** [comment]
>I'm not implementing MemoryFilter. That makes compatibility lower.

You can check addresses to prevent some of the problems. I think that would work for games like F-Zero. I'm not exactly saying MemoryFilter is necessary, just that something needs to be done, in order to improve game compatibility.

**Comment by cxd4** [comment]
I think you mean "all" of the problems.

If MemoryFilter was a requirement to compatible N64 emulation, then an Android port would be impossible.

Now, the reason I don't have any idea what you said.  Is because I have _never_ seen any error message like that.  Any game I run, runs perfectly without errors.  If you disagree, you need to either a) report that or b) file a pull request to fix it.  Because I don't have whatever rom you are thinking of.

**Comment by LegendOfDragoon** [comment]
>Now, the reason I don't have any idea what you said. Is because I have never seen any error message like that. Any game I run, runs perfectly without errors. If you disagree, you need to either a) report that or b) file a pull request to fix it. Because I don't have whatever rom you are thinking of.

I understand that we may be testing different games. Most games work. Some work depending on whether you enabled expansion pack or not. For instance, SD Hiryuu no Ken Densetsu gets an Unhandled Memory Action error, only if you're using 4MB. Some of the issues I encountered might be my fault (like WDC not working in 64bit). I'm probably going to have to examine the code carefully.

**Comment by cxd4** [comment]
I'll commit the RDB file in so you can PR anything that's just because of using 4 MB.

What happens is while a game usually only cares about 4 MB, it's welcome to attempting testing to see how much RAM there is anyway to use extra features.  Those are the games that should have those enabled, so it should be 8 or 16 MB set for those games.

**Comment by LegendOfDragoon** [comment]
Alright. Looks like the WDC issue was indeed a mistake on my part. I forgot that I was using Azimer's 0.55 which forces audio sync and its audio sync makes WDC run super slow. I got it working with Azimer's XA2 WIP5. I may try seeing if I can fix the UI bugs in 64-bit PJ64 2.2.

Dezaemon 3D (J) is another game that has an Unhandled Memory Action, regardless of expansion pack being enabled or not.

**Comment by cxd4** [comment]
>Dezaemon 3D (J) is another game that has an Unhandled Memory Action, regardless of expansion pack being enabled or not.

Create an issue on my Project64_64 repo so we can continue the discussion there, as this will probably confuse the people reading this page who think this may apply to official, upstream Project64 32-bit.

**Comment by MatthewHunter1983** [comment]
rofl XD thanks guys. funny I was just thinking before I checked here again right now, hmmm I wonder if the 64 bit build is new.

**Comment by cxd4** [comment]
>I wonder if the 64 bit build is new.

Yep.  That stuff was decided not too long ago by later changes zilmar did.
